scripts/get_all_heron_paths.sh

```diff
@@ -23,7 +23,19 @@ function query() {
 
 set +e
 # Build everything
-bazel build {heron,integration-test,tools/java}/...
+os_type=darwin
+case "$OSTYPE" in
+  darwin*)
+    os_type=darwin
+    ;;
+  linux*)
+    os_type=ubuntu
```

Comment:
how could you distinguish between different flavors of linux? We support both centos and ubuntu.